src/components/20-molecules/dropdown/react/DemoUncontrolledDropdownReact.jsx

```diff
@@ -3,28 +3,40 @@ import React from 'react';
 import AXADropdownReact from './AXADropdownReact';
 import AXATextReact from './AXATextReact';
 
-const DemoUncontrolledDropdown = props => {
+function DemoUncontrolledDropdown({
+  item1,
+  item2,
+  item3,
+  label,
+  defaultTitle,
+  name,
+  invalid,
+  error,
+  native,
+  required,
+  checkMark,
+  disabled,
+}) {
   const items = [
-    { name: props.item1, value: 'Item 1' },
-    { name: props.item2, value: 'Item 2', selected: true },
-    { name: props.item3, value: 'Item 3' },
+    { name: item1, value: 'Item 1' },
+    { name: item2, value: 'Item 2', selected: true },
+    { name: item3, value: 'Item 3' },
   ];
 
   return (
     <>
       <AXADropdownReact
         data-test-id="uncontrolled-dropdown-react"
         items={items}
-        label={props.label}
-        defaultTitle={props.defaultTitle}
-        name={props.name}
-        invalid={props.invalid}
-        error={props.error}
-        native={props.native}
-        required={props.required}
-        checkMark={props.checkMark}
-        disabled={props.disabled}
-        // eslint-disable-next-line no-console
+        label={label}
```

Comment:
You could destructure `props` at line 20 or so, and still use the simpler elements like you do here.